src/bcccp/carpark/CarSensor.java

```diff
@@ -99,6 +99,8 @@ public boolean carIsDetected() {
 		return carDetected;
 	}
 	
-	
+	public void setCarDetected(Boolean carDetected) {
```

Comment:
Do you have a way to un-set this?
maybe something like carDetected = !carDetected 
that way if you wantted to simulate a car driving up to the gate, then driving away you could

Reply:
Can just set the parameter to false, don't really see the issue here.

Reply:
my bad, no excuse for that, sorry